root/layout/index.js

```diff
@@ -88,6 +97,22 @@ const Layout = ({$c, ...props}) => (
           </p>
         </div>}
 
+      {$c.user &&
+       $c.user.birth_date &&
+       moment($c.user.birth_date).month() === moment().month() &&
```

Comment:
If you change the `birth_date` serialization above, we can do `$c.user.birth_date.month === new Date().getMonth() + 1` here (+ 1 because the months are 0-indexed), and `$c.user.birth_date.day === new Date().getDate()` (no + 1 because the day is 1-indexed, because JavaScript), which is much faster than using moment.